interop-testing/src/test/java/io/grpc/testing/integration/ProxyTest.java

```diff
@@ -135,11 +144,17 @@ public void smallBandwidth() throws Exception {
 
     clientOut.write(new byte[1]);
     clientIn.readFully(new byte[100 * 1024]);
-    long start = System.nanoTime();
-    clientIn.readFully(new byte[5 * bandwidth]);
-    long stop = System.nanoTime();
-
-    long bandUsed = ((5 * bandwidth) / ((stop - start) / TimeUnit.SECONDS.toNanos(1)));
+    int sample = bandwidth / 5;
+    List<Double> bandwidths = new ArrayList<>();
+    for (int i = 0; i < 5; i++) {
+      long start = System.nanoTime();
+      clientIn.readFully(new byte[sample]);
```

Comment:
The reason that "bandwidth" was chosen here is because it was roughly "1 second worth of data". In order to prevent this test from taking 5 seconds, I do 5 tests each 1/5 the size. I could make 'int trials = 5' variable if that would make it more clear.